.github/CODEOWNERS

```diff
@@ -48,7 +48,6 @@ yarn.lock                                           @backstage/maintainers @back
 /plugins/events-backend-module-gerrit               @backstage/maintainers @pjungermann
 /plugins/events-backend-module-github               @backstage/maintainers @pjungermann
 /plugins/events-backend-module-gitlab               @backstage/maintainers @pjungermann
-/plugins/events-backend-test-utils                  @backstage/maintainers @pjungermann
 /plugins/events-node                                @backstage/maintainers @pjungermann
```

Comment:
as the new `/packages/events` content is an extraction of parts of `/plugins/events-node`, you can put/keep me as one of the CODEOWNERS.